.gitattributes

```diff
@@ -3,3 +3,8 @@
 # means that when checking out code, LF endings get converted
 # to CRLF.
 # *.svg text eol=lf
+*.rb   diff=ruby
+*.rake diff=ruby
+*.css  diff=css
+*.html diff=html
+*.erb  diff=html
```

Comment:
there's a missing newline at the end of the file

Reply:
> Hi @Rafi993, thanks for the change.
> 
> What about frontend files? JS, JSX and so on? Should we add those as well

They don't seem to have support for js or ts https://github.com/git/git/blob/master/userdiff.c#L78  as mentioned in this thread https://gist.github.com/tekin/12500956bd56784728e490d8cef9cb81

Reply:
> there's a missing newline at the end of the file

I'll add the newline now.

Reply:
> They don't seem to have support for js or ts https://github.com/git/git/blob/master/userdiff.c#L78 as mentioned in this thread https://gist.github.com/tekin/12500956bd56784728e490d8cef9cb81

got it, thanks!